go.mod

```diff
@@ -5,6 +5,10 @@ module nimona.io
 require (
 	github.com/cheekybits/genny v1.0.0
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/docker v1.13.1
```

Comment:
Can we please move these inside `/internal/simulation` and make it a standalone package?